pic_database.php

```diff
@@ -0,0 +1,61 @@
+<?php
+	include('config/config.php');
+	include('include/db_query.php');
+
+	// InsertPic('Wes Hicks', '2018-05-25', 'TestPic', 'pictures/0f191f-wes-hicks-464614-unsplash.jpg', 'flavortext');
+	$pic = GetPic($_REQUEST['picID']);
+	//var_Dump($pic);
+	DisplayPic($pic);
+	// echo"<img src='pictures/0f191f-wes-hicks-464614-unsplash.jpg'alt='".$pic['flavor']."'>";
+
+
+	function InsertPic($photographer, $date, $title, $link, $flavor){
```

Comment:
I'm not 100% sure what this file is for, but I think you'll want to make a file in your /include folder with all these functions. I'd suggest putting almost every function you write in the include folder, and then include that file in your web pages (view_post, index, etc.).

Check out how I structured it in my example code: https://github.com/tilikang/coding_fellowship_2018/tree/sample-blog-no-db

Look at view_post.php.  See how it includes "include_all.php" at the top? Look at that file and you'll see that it includes all my other files. By taking this approach, I always have access to all of the functions in all of my code with just one include at the top of each web page.